src/controllers/applicant.ts

```diff
@@ -8,13 +8,15 @@ export const createApplicant = async (req: Request, res: Response): Promise<void
         _id: new mongoose.Types.ObjectId(),
         firstName: req.body.firstName,
         lastName: req.body.lastName,
+        email: req.body.email,
         role: req.body.role,
         major: req.body.major,
         yearStanding: req.body.yearStanding,
         status: req.body.status,
         linkedIn: req.body.linkedIn,
         website: req.body.website,
         resume: req.body.resume,
+        sheetURL: req.body.sheetURL,
```

Comment:
Looks like this is a singular reference to a sheet - could we make this an array of ObjectId for each of the sheet elements? Makes sense later on if we would like to link multiple sheets to a single user